torch/fx/experimental/symbolic_shapes.py

```diff
@@ -4257,10 +4257,15 @@ def create_symbol(
                 sympy_expr
             ) in config.extended_debug_create_symbol.split(",")
             maybe_more_info = ""
-            if not is_debug:
+            if (
+                not is_debug
+                and os.getenv("TORCHDYNAMO_EXTENDED_ADVICE=0") is not None
```

Comment:
this is wrong

Reply:
@ezyang ah, it should be
`and os.getenv("TORCHDYNAMO_EXTENDED_ADVICE=0") is None`
right?

Or do you mean, that the naming is still incorrect?

Reply:
don't pass =0 to the getenv. Grep for use of getenv in codebase, it will show the right pattern

Reply:
@ezyang 
Got you. The current behavior is to show the extended info, so I would take that as default and use the following condition:
`if not is_debug and os.getenv("TORCHDYNAMO_EXTENDED_ADVICE", "1") != "0"`

To me it seems correct that there is extended info if `TORCHDYNAMO_EXTENDED_ADVICE=1` and no extended info if `TORCHDYNAMO_EXTENDED_ADVICE=0`.

The user can then suppress the extended info by setting `TORCHDYNAMO_EXTENDED_ADVICE=0`. This will be noted in the `maybe_more_info` below.

Let me know if you agree, if so, I will make and commit the changes.

Reply:
Yes it defaults to 1. You may want to also handle empty string too.

Reply:
@ezyang Thanks, I made the changes and committed them. I know that this is a PR for a small issue and I appreciate the support and time taken to review.